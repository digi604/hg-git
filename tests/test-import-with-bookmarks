#!/bin/sh

"$TESTDIR/hghave" git || exit 80

echo "[extensions]" >> $HGRCPATH
echo "bookmarks = " >> $HGRCPATH

hg init
git init -q

git fast-import --quiet < "$TESTDIR/git-sample-repo"

hg init

echo % import with bookmarks
hg gimport
hg log

echo % we should have some bookmarks
hg book
