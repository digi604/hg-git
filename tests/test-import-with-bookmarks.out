abort: repository . already exists!
% import with bookmarks
importing Git objects into Hg
at: 0/2
changeset:   1:c25ea3d53ce1
tag:         master
tag:         tip
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add beta

changeset:   0:c50ac8b9444c
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add alpha

% we should have some bookmarks
   master                    1:c25ea3d53ce1
